### DISCUSSION TITLE
EngineDataLogger - Beginners question related to Wiki

### DISCUSSION THREAD

**Discussion by kurt-k3po** [techie66/EngineDataLogger#19] [Q&A] [answered]
Hallo Jacob,
if I understand correctly, download and compilation takes place directly on the Raspberry, right? No other development environment on separate Linux box is used?

regards
Kurt

**Comment by techie66** [comment, marked as answer]
Yes, that is correct.

**Comment by kurt-k3po** [comment]
O.k. now I finished setup the Pi Zero W from scratch following the 'Getting started' in the Wiki.

I struggled a bit as 'cmake' and 'autoreconf' are not installed by default oder by installing 'built-essential'
'cmake' is called from 'install-deps.sh flatbuffers' and 'autoreconf' when installing 'isp'.
Install them with:
       $ sudo apt install dh-autoreconf cmake

My suggestion is, either mention that in the 'Getting started' section of the Wiki, or modify the install-deps.sh script acordingly.

regards
Kurt

**Reply by techie66** [reply]
I think both is a good idea. it should be mentioned in the wiki, but I will have the install-deps.sh do it automatically as well
